src/components/institution_table/InstitutionTable.js

```diff
@@ -72,6 +74,14 @@ export default function InstitutionTable(props) {
       div({style: Styles.TABLE.CONTAINER}, [
         div({style: Styles.TABLE.HEADER_ROW}, [tableHeaderTemplate]),
         filteredList.slice((currentPage - 1) * tableSize, (currentPage * tableSize)).map((inst, index) => {
+          let signingOfficials = '';
+          isNil(inst.signingOfficials) ? signingOfficials = '' : inst.signingOfficials.forEach((user, i) => {
+            signingOfficials = signingOfficials.concat(user.displayName, ' (', user.email, ')');
+            //if there are multiple SOs in the list and this is not the last one, add comma
+            if (inst.signingOfficials.length > 1 && i < (inst.signingOfficials.length - 1)) {
+              signingOfficials = signingOfficials.concat(', ');
+            }
```

Comment:
Another way to approach this is to generate an array of strings, each one representing an SO. Then, just join the array with the `,` and you'll get the same effect as using an indexed loop.